### ISSUE TITLE
[Testing] Make integration test output readable

### ISSUE THREAD

**Issue by decentralgabe** [TBD54566975/ssi-service#436]
Right now it looks something like this

```
time="2023-05-18T01:40:02Z" level=info msg="\n\nCreate a verifiable credential"
time="2023-05-18T01:40:02Z" level=info msg="\nPerforming PUT request to:  http://localhost:8080/v1/credentials \n\nwith data: \n{\"data\":{\"givenName\":\"ricky bobby\",\"additionalName\":\"hank hill\",\"familyName\":\"simpson\",\"birthDate\":\"2009-01-03\",\"postalAddress\":{\"addressCountry\":\"U.S.A\",\"addressLocality\":\"Austin\",\"addressRegion\":\"TX\",\"postalCode\":\"78724\",\"streetAddress\":\"123 Janktopia Ave.\"},\"taxID\":\"123\"},\"issuer\":\"did:ion:EiBoq_1MmikChd-lWKe-tRDmOTQrsT_usYWMPKc2s0VjvQ\",\"issuerKid\":\"5fda2ffd-f9c3-4de8-bf08-7770ca63edfe\",\"subject\":\"did:ion:EiBoq_1MmikChd-lWKe-tRDmOTQrsT_usYWMPKc2s0VjvQ\",\"@context\":\"[https://www.w3.org/2018/credentials/v1\](https://www.w3.org/2018/credentials/v1/)",\"expiry\":\"2051-10-05T14:48:00.000Z\",\"schemaId\":\"a533e533-7e15-4c1e-ab75-9d9121bcac88\"}\n"
time="2023-05-18T01:40:02Z" level=info msg="\nOutput:"
time="2023-05-18T01:40:02Z" level=info msg="{\n    \"credential\": {\n        \"@context\": [\n            \"[https://www.w3.org/2018/credentials/v1\](https://www.w3.org/2018/credentials/v1/)"\n        ],\n        \"id\": \"78ed6e5c-34a7-43b4-a4e1-19574b666497\",\n        \"type\": [\n            \"VerifiableCredential\"\n        ],\n        \"issuer\": \"did:ion:EiBoq_1MmikChd-lWKe-tRDmOTQrsT_usYWMPKc2s0VjvQ\",\n        \"issuanceDate\": \"2023-05-18T01:40:02Z\",\n        \"expirationDate\": \"2051-10-05T14:48:00.000Z\",\n        \"credentialSubject\": {\n            \"additionalName\": \"hank hill\",\n            \"birthDate\": \"2009-01-03\",\n            \"familyName\": \"simpson\",\n            \"givenName\": \"ricky bobby\",\n            \"id\": \"did:ion:EiBoq_1MmikChd-lWKe-tRDmOTQrsT_usYWMPKc2s0VjvQ\",\n            \"postalAddress\": {\n                \"addressCountry\": \"U.S.A\",\n                \"addressLocality\": \"Austin\",\n                \"addressRegion\": \"TX\",\n                \"postalCode\": \"78724\",\n                \"streetAddress\": \"123 Janktopia Ave.\"\n            },\n            \"taxID\": \"123\"\n        },\n        \"credentialSchema\": {\n            \"id\": \"a533e533-7e15-4c1e-ab75-9d9121bcac88\",\n            \"type\": \"JsonSchemaValidator2018\"\n        }\n    },\n    \"credentialJwt\": \"***"\n}"
time="2023-05-18T01:40:02Z" level=info msg="\n\nCreate our Credential Manifest:"
time="2023-05-18T01:40:02Z" level=info msg="\nPerforming PUT request to:  http://localhost:8080/v1/manifests \n\nwith data: \n{\"issuerDid\":\"did:ion:EiBoq_1MmikChd-lWKe-tRDmOTQrsT_usYWMPKc2s0VjvQ\",\"issuerKid\":\"5fda2ffd-f9c3-4de8-bf08-7770ca63edfe\",\"issuerName\":\"Issuer Name Lol\",\"format\":{\"jwt\":{\"alg\":[\"EdDSA\"]}},\"outputDescriptors\":[{\"id\":\"kyc_credential\",\"schema\":\"a533e533-7e15-4c1e-ab75-9d9121bcac88\"}],\"presentationDefinition\":{\"id\":\"32f54163-7166-48f1-93d8-ff217bdb0653\",\"name\":\"KYC Requirements\",\"purpose\":\"TBD\",\"format\":{\"jwt\":{\"alg\":[\"EdDSA\"]}},\"input_descriptors\":[{\"id\":\"kyc1\",\"name\":\"Personal Info\",\"constraints\":{\"subject_is_issuer\":\"required\",\"fields\":[{\"id\":\"givenName\",\"path\":[\"$.vc.credentialSubject.givenName\"],\"filter\":{\"type\":\"string\",\"pattern\":\"[a-zA-Z \\\\-\\\\.].+\"}},{\"id\":\"additionalName\",\"path\":[\"$.vc.credentialSubject.additionalName\"],\"filter\":{\"type\":\"string\",\"pattern\":\"[a-zA-Z \\\\-\\\\.].+\"}},{\"id\":\"familyName\",\"path\":[\"$.vc.credentialSubject.familyName\"],\"filter\":{\"type\":\"string\",\"pattern\":\"[a-zA-Z \\\\-\\\\.].+\"}},{\"id\":\"birthDate\",\"path\":[\"$.vc.credentialSubject.birthDate\"],\"filter\":{\"type\":\"string\",\"format\":\"date\"}},{\"id\":\"taxID\",\"path\":[\"$.vc.credentialSubject.taxID\"],\"filter\":{\"type\":\"string\"}},{\"id\":\"postalAddress\",\"path\":[\"$.vc.credentialSubject.postalAddress\"],\"filter\":{\"type\":\"object\",\"properties\":{\"addressCountry\":{\"type\":\"string\"},\"addressLocality\":{\"type\":\"string\"},\"addressRegion\":{\"type\":\"string\"},\"postalCode\":{\"type\":\"string\"},\"streetAddress\":{\"type\":\"string\"}},\"required\":[\"addressCountry\",\"addressLocality\",\"addressRegion\",\"postalCode\",\"streetAddress\"]}}]}}]}}\n"
time="2023-05-18T01:40:02Z" level=info msg="\nOutput:"
time="2023-05-18T01:40:02Z" level=info msg="{\n    \"credential_manifest\": {\n        \"id\": \"0dab3312-80ed-4b13-ad7f-eb9a8309d18b\",\n        \"spec_version\": \"[https://identity.foundation/credential-manifest/spec/v1.0.0/\](https://identity.foundation/credential-manifest/spec/v1.0.0//)",\n        \"issuer\": {\n            \"id\": \"did:ion:EiBoq_1MmikChd-lWKe-tRDmOTQrsT_usYWMPKc2s0VjvQ\",\n            \"name\": \"Issuer Name Lol\"\n        },\n        \"output_descriptors\": [\n            {\n                \"id\": \"kyc_credential\",\n                \"schema\": \"a533e533-7e15-4c1e-ab75-9d9121bcac88\"\n            }\n        ],\n        \"format\": {\n            \"jwt\": {\n                \"alg\": [\n                    \"EdDSA\"\n                ]\n            }\n        },\n        \"presentation_definition\": {\n            \"id\": \"32f54163-7166-48f1-93d8-ff217bdb0653\",\n            \"name\": \"KYC Requirements\",\n            \"purpose\": \"TBD\",\n            \"format\": {\n                \"jwt\": {\n                    \"alg\": [\n                        \"EdDSA\"\n                    ]\n                }\n            },\n            \"input_descriptors\": [\n                {\n                    \"id\": \"kyc1\",\n                    \"name\": \"Personal Info\",\n                    \"constraints\": {\n                        \"fields\": [\n                            {\n                                \"id\": \"givenName\",\n                                \"path\": [\n                                    \"$.vc.credentialSubject.givenName\"\n
```

this makes it very hard to determine what is happening, and what is failing.

**Comment by decentralgabe** [comment]
To reproduce you need to have a server running, and run integration tests. There are a few ways to do this.

The simplest way is to go to the `build` folder and run `docker-compose build && docker-compose up` followed by, from the root directory of the repo, running `mage integration`.

Importantly the log level needs to be set to `info` or below. I recommend, for debugging, setting the output to `debug` or `trace`.

**Comment by decentralgabe** [comment]
fixed in #444
